hermetic_build/README.md

```diff
@@ -18,20 +18,36 @@ for additional instructions.
 
 ## Prerequisites
 
-In order to generate a version for each library, a versions.txt has to exist
+### Generation configuration
+
+A generation configuration file is required to generate GAPIC Client Libraries.
+
+Please refer to [Generation configuration yaml](#generation-configuration-yaml--generation-config-path---optional)
+for more information.
+
+### versions.txt
```

Comment:
IIRC, we would generate a `versions.txt` file if it does not exist. Can you confirm @diegomarquezp ?

Reply:
No, it is not generated. The scripts will just fail if not found.

https://github.com/googleapis/sdk-platform-java/blob/1080de4f1d437d7314455f91cfd674220fb866d8/hermetic_build/library_generation/utils/utilities.py#L124-L142

Reply:
I see, thanks for the quick response!

Reply:
The file is required, per https://github.com/googleapis/sdk-platform-java/blob/7d6d66a161db5edc538aec065405954acf4434c5/hermetic_build/library_generation/utils/utilities.py#L164

Though I think we can create an empty file if not exist since we didn't check the contents.

I'll create a follow-up issue.